src/resolvers/Topics.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const Promise = require('bluebird');
```

Comment:
You're right, I don't need it here. I do need to include bluebird to use its promisifyAll. Files like settings.js and blobstoragemanager.js need bluebird.